opentreemap/importer/tasks.py

```diff
@@ -144,7 +144,10 @@ def commit_import_event(import_type, import_event_id):
 
     finalize_task = _finalize_commit.si(import_type, import_event_id)
 
-    chord(commit_tasks, finalize_task).delay()
+    async_result = chord(commit_tasks, finalize_task).delay()
```

Comment:
`AsyncResult` does not have a save method. I originally tried to use a `GroupResult` for this feature and was not able to make it work.